apps/src/code-studio/components/progress/MiniViewTopRow.story.jsx

```diff
@@ -7,6 +7,12 @@ import progress from '@cdo/apps/code-studio/progressRedux';
 export default storybook => {
   const initialState = {
     progress: {
+      stages: [
+        {
+          levels: []
+        }
+      ],
+      focusAreaStageIds: [],
```

Comment:
Sorry for the bad advice -- looks like i only searched `code-studio/components`.